wiz/pipeline.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: "Scan different Directory"
```

Comment:
Is this label right? This file is the one that will be rendered on the template visualisation. So perhaps something like `:wiz: Scan project` might be clearer? 

e.g. 

<img width="767" alt="Screenshot 2024-07-03 at 10 35 39 am" src="https://github.com/buildkite/templates/assets/656826/f1ed06df-4970-4ee7-b9bd-cd453b8beedc">